### PR TITLE
:sparkles: [#483] add description for radio en checkbox option

### PR DESCRIPTION
**Pull request by nikkiysendoorn1**
Closes open-formulieren/open-forms-sdk#483

### CODE REVIEW

**Review by Viicos — COMMENTED** [review]
If I remember correctly, the issue was to add a description for checkboxes in the SDK (that is, available to the end user). In the form builder (this repo), we actually need to add a text input for each option where the description, so probably a 3rd column here:

image

? Maybe we should confirm with @joeribekker

**Review by sergei-maertens — COMMENTED** [review]
Couple thoughts/remarks:

* the description takes up too much space. Rather, make this "second row" collapsed by default, and add a link/icon to "edit option description" which expands/collapses it on click
* I can't seem to enter translations yet for these descriptions :)

File: src/components/builder/values/option-row.tsx [hunk 1]
```diff
@@ -63,69 +64,87 @@ const OptionRow: React.FC<OptionRowProps> = ({name, index, arrayHelpers}) => {
...
+          <FormattedMessage
+            description="Option label table header/label"
+            defaultMessage="Description"
+          />
+          <TextArea
+            name={`${fieldNamePrefix}.description`}
+            aria-label={intl.formatMessage({
+              description: 'Accessible label for option label',
+              defaultMessage: 'Option description',
+            })}
```
Comment (sergei-maertens) [inline src/components/builder/values/option-row.tsx:144]:
```suggestion
            aria-label={intl.formatMessage({
              description: 'Accessible label for option description',
              defaultMessage: 'Option description',
            })}
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
What I meant was that when I enter the description of an option, there's no way to go to the "translations" tab and enter the Dutch/English translation for that same description.

If a form is being filled out in English, the option labels (and descriptions) must display their english translations instead of the default/Dutch translations. Currently the person defining the form cannot enter these translations anywhere.

**Review by sergei-maertens — APPROVED** [review]
I've done some more UI tweaks :)

**Pull request merged**